src/overlays/actors/ovl_En_Bom_Chu/z_en_bom_chu.c

```diff
@@ -336,22 +359,26 @@ void EnBomChu_WaitForKill(EnBomChu* this, GlobalContext* globalCtx) {
     }
 }
 
-void func_809C649C(EnBomChu* this, Vec3f* arg1, Vec3f* dest) {
-    f32 scale = arg1->x + this->unk_178;
-
-    dest->x =
-        this->actor.world.pos.x + (this->unk_16C.x * scale) + (this->unk_160.x * arg1->y) + (this->unk_154.x * arg1->z);
-    dest->y =
-        this->actor.world.pos.y + (this->unk_16C.y * scale) + (this->unk_160.y * arg1->y) + (this->unk_154.y * arg1->z);
-    dest->z =
-        this->actor.world.pos.z + (this->unk_16C.z * scale) + (this->unk_160.z * arg1->y) + (this->unk_154.z * arg1->z);
+/*
+ * Transform coordinates from model space to world space, according to current orientation.
+ * `posModel` is expected to already be at world scale (1/100 compared to model scale)
+ */
+void EnBomChu_ModelToWorld(EnBomChu* this, Vec3f* posModel, Vec3f* dest) {
+    f32 x = posModel->x + this->visualJitter;
+
+    dest->x = this->actor.world.pos.x + (this->axisLeft.x * x) + (this->axisUp.x * posModel->y) +
+              (this->axisForwards.x * posModel->z);
+    dest->y = this->actor.world.pos.y + (this->axisLeft.y * x) + (this->axisUp.y * posModel->y) +
+              (this->axisForwards.y * posModel->z);
+    dest->z = this->actor.world.pos.z + (this->axisLeft.z * x) + (this->axisUp.z * posModel->y) +
+              (this->axisForwards.z * posModel->z);
 }
 
-void EnBomChu_SpawnRipples(EnBomChu* this, GlobalContext* globalCtx, f32 height) {
+void EnBomChu_SpawnRipples(EnBomChu* this, GlobalContext* globalCtx, f32 y) {
     Vec3f pos;
 
     pos.x = this->actor.world.pos.x;
-    pos.y = height;
+    pos.y = y;
```

Comment:
personally like height more in this context, but am biased cause im the one that called it that originally lol

Reply:
I would agree with height if it was relative to some coordinate, usually "height" refers to a (positive) vertical distance between two points